Content.Server/Singularity/Components/EventHorizonComponent.cs

```diff
@@ -0,0 +1,33 @@
+using Content.Shared.Singularity.Components;
+using Content.Server.Singularity.EntitySystems;
+
+namespace Content.Server.Singularity.Components;
+
+/// <summary>
+/// The server-side version of <see cref="SharedEventHorizonComponent">.
+/// Primarily managed by <see cref="EventHorizonSystem">.
+/// </summary>
+[RegisterComponent]
+[ComponentReference(typeof(SharedEventHorizonComponent))]
+public sealed class EventHorizonComponent : SharedEventHorizonComponent
```

Comment:
You can just get rid of the server & client components and have a single (shared) `EventHorizonComponent`

Reply:
Do we want the client to know about pulse timings? Looking at the old singularity component that looks like a server-side thing.

Reply:
Done